WordPress/gutenberg
# Update the Modal Heading and Label text color value

Update the Modal Heading and Label text color value so that it matches with the text color used for other areas of the editor. Previously it was using color from wp-admin common.css. Fixes #50448



## What?


## Why?


## How?


## Testing Instructions
1. Open the WordPress page editor.2. 
2. Open the Preferences modal and check the colour value using inspect element.

### Testing Instructions for Keyboard


## Screenshots or screencast 
<img width="975" alt="Screenshot 2023-05-24 at 2 56 21 PM" src="https://github.com/WordPress/gutenberg/assets/7237291/daadc4e4-4e4e-4410-8b2a-b53f7508effd">
<img width="721" alt="Screenshot 2023-05-24 at 2 55 49 PM" src="https://github.com/WordPress/gutenberg/assets/7237291/e56cdc57-7ed4-4d7b-a9f7-af750f8e05d8">
